### DISCUSSION TITLE
date to not be in UTC

### DISCUSSION THREAD

**Discussion by acollazomayer** [henninghall/react-native-date-picker#418] [Q&A]
is there a way to not get the date as UTC in the onChangeEvent? Thanks

**Comment by TalhaAbid1** [comment]
### @acollazomayer I Found A Way To Change This
Here is the Solution:
***Just Simply Go To***
> node_modules\react-native-date-picker\android\src\main\java\com\henninghall\date_picker\propsUtcProp.java
> On Line No 6 You Can Change It Accordingly

Save It & Rebuild Your Project

```javascript
npx react-native run-android
npx react-native run-ios
